jsonnetfile.lock.json

```diff
@@ -8,8 +8,8 @@
           "subdir": "grafana"
         }
       },
-      "version": "539a90dbf63c812ad0194d8078dd776868a11c81",
```

Comment:
Does this update everything?

Reply:
Can you elaborate on "everything"?

Reply:
e.g. did you run jb update and commit all the deps, or just parts, as I think now would be a good time.

Reply:
I did it in stages to help with verifying what is changing, but in the end, this came down to `jb update`.

Reply:
Yay 🎉